libraries/chain/controller.cpp

```diff
@@ -3120,11 +3121,6 @@ block_state_ptr controller::fetch_block_state_by_number( uint32_t block_num )con
 } FC_CAPTURE_AND_RETHROW( (block_num) ) }
 
```

Comment:
Maybe add a comment saying this is thread-safe.

Reply:
There is a comment in the header that says that.